operations/kubernetes/Backups_for_etcd-operator_Clusters.md

```diff
@@ -16,65 +18,99 @@ The following services are backed up daily \(one week of backups retained\) as p
 - Firmware Action Service \(FAS\)
 - User Access Service \(UAS\)
 
-(`ncn-mw#`) Run the following command on any Kubernetes master or worker node in order to list the backups for a specific project.
-In the example below, the backups for BSS are listed.
+If these clusters are lacking a recent backup, create a manual backup for that service by following the
+[Create a Manual Backup of a Healthy etcd Cluster](Create_a_Manual_Backup_of_a_Healthy_etcd_Cluster.md) procedure.
+
+## Clusters without automated backups
+
+The following projects are not backed up as part of the automated solution:
```

Comment:
I know you only moved this, but "projects" is probably not the right word. Above, we use "services", which I think still makes sense here:

```suggestion
The following services are not backed up as part of the automated solution:
```

Reply:
fixed